plctestbench/data_manager.py

```diff
@@ -131,7 +139,10 @@ def _save_run_to_database(self):
         self.run['creator'] = self.user['email']
         self.run['created_on'] = datetime.datetime.now()
         self.run['status'] = 'CREATED'
+        self.run['selected_input_files'] = list(map(lambda node: os.path.basename(node.file.path), self.root_nodes))
```

Comment:
This information is already present in the database. The "workers" array in the runs collection stores 4 lists. The first one contains the settings for the OriginalAudio workers, there you can find the filename of the selected input files

Reply:
![screenshot](https://github.com/LucaVignati/plc-testbench/assets/17725043/87801f92-7c04-4357-805e-62c6bf5fd2d7)

Reply:
<img width="323" alt="Screenshot 2023-08-11 at 20 29 43" src="https://github.com/LucaVignati/plc-testbench/assets/17725043/62c521d0-aded-498a-bc99-e6e8a97163b9">